docs/releasing.md

```diff
@@ -2,41 +2,82 @@
 
 Steps for a new release (these are in the process of being automated):
 
-1. Check out `main` and make sure you have pulled down the latest changes (`git pull origin main --tags`)
+1. Check out `main` and make sure you have pulled down the latest changes and tags
+   
+   ```
+   git checkout main
+   git pull --tags
+   ```
 
-2. Run `yarn release` which uses [`standard-version`](https://github.com/conventional-changelog/standard-version) to:
+2. Determine the version number of the new release
+   
+   ```
+   yarn release --dry-run
```

Comment:
Nice yes -  this is good to have as just the way we should release. Also good way to learn about whats happening